package.json

```diff
@@ -1,49 +1,27 @@
 {
   "name": "divviup-ts",
-  "version": "0.1.0",
-  "description": "",
-  "main": "src/index.ts",
+  "workspaces": [
+    "./packages/common",
+    "./packages/field",
+    "./packages/prg",
+    "./packages/vdaf",
+    "./packages/prio3",
+    "./packages/dap"
+  ],
   "scripts": {
-    "build": "tsc",
-    "docs": "typedoc src",
-    "test": "ts-mocha \"src/**/*.spec.ts\" --paths src -p ./tsconfig.json",
-    "lint": "eslint . --ext .ts && prettier -c src",
-    "format": "prettier -w src",
-    "test:coverage": "nyc --reporter=lcov --reporter=html --reporter=text --reporter=text-summary npm test",
-    "ci": "npm run lint && npm run test:coverage",
-    "benchmark": "ts-node scripts/benchmarks.ts"
-  },
-  "license": "MPL-2.0",
-  "dependencies": {
-    "@guildofweavers/galois": "^0.4.22",
-    "aes-cmac": "^1.0.3",
-    "hpke": "^0.2.1",
-    "undici": "^5.8.1"
+    "test": "lerna run test",
+    "build": "lerna run build",
+    "test:coverage": "rm -rf coverage && rm -rf .nyc_output && lerna run test:coverage",
+    "lint": "lerna run lint"
   },
   "devDependencies": {
-    "@istanbuljs/nyc-config-typescript": "^1.0.2",
-    "@tsconfig/node18": "^1.0.1",
-    "@types/benchmark": "^2.1.1",
-    "@types/expect": "^24.3.0",
-    "@types/express": "^4.17.13",
-    "@types/mocha": "^9.1.1",
-    "@types/node": "^18.6.5",
-    "@typescript-eslint/eslint-plugin": "^5.27.0",
-    "@typescript-eslint/parser": "^5.29.0",
-    "benchmark": "^2.1.4",
-    "eslint": "^8.16.0",
-    "eslint-config-prettier": "^8.5.0",
-    "express": "^4.18.1",
-    "microtime": "^3.1.0",
-    "mocha": "^10.0.0",
-    "nyc": "^15.1.0",
-    "prettier": "^2.6.2",
-    "source-map-support": "^0.5.21",
-    "ts-mocha": "^10.0.0",
-    "ts-node": "^10.8.1",
-    "tsconfig-paths": "^4.0.0",
-    "typedoc": "^0.23.10",
-    "typescript": "^4.7.4",
-    "typescript-eslint-language-service": "^5.0.0"
+    "@parcel/packager-ts": "^2.7.0",
```

Comment:
We should add `parcel` as a top-level devDependency as well. I tried running `npm i && npm run build`, and it failed on the `parcel build` command prior to installing it.